.readthedocs.yaml

```diff
@@ -9,19 +9,20 @@ version: 2
 build:
   os: ubuntu-22.04
   tools:
-    # updated and deployed from Aug 1, 2023
-    python: "mambaforge-22.9"
-  jobs:
-    pre_create_environment:
-      # update mamba just in case
-      - mamba update --yes --quiet --name=base mamba 'zstd=1.5.2'
-      - mamba --version
-      - mamba list --name=base
-    post_create_environment:
-      - conda run -n ${CONDA_DEFAULT_ENV} mamba list
-      # use conda run executable wrapper to have all env variables
-      - conda run -n ${CONDA_DEFAULT_ENV} mamba --version
-      - conda run -n ${CONDA_DEFAULT_ENV} pip install . --no-deps
```

Comment:
It looks like the build is failing because ESMValTool is not installed because this line was removed

Reply:
ha! This shows how much attention I was paying at this on a Friday at 5pm - cheers, bud, I'll restore it :beer:

Reply:
OK buds I put that all-important line back in 42c8b4fa3ca6e4ebd95f6e703db67536842e161e and now the docs fail with the customary issue seen before (from the config changes), I need to put in a comment in the RTD conf file, merge `main` and this one's ready for shipping :ship: `minoconda` back in makes the build take about 500s so it's not THAT bad compared to Mambaforge3